README.md

```diff
@@ -201,23 +201,15 @@ You can configure Neovim to automatically run `:PackerCompile` whenever `plugins
 [an autocommand](https://neovim.io/doc/user/autocmd.html#:autocmd):
 
 ```
-augroup packer_user_config
-  autocmd!
-  autocmd BufWritePost plugins.lua source <afile> | PackerCompile
-augroup end
+local group = vim.api.nvim_create_augroup("packer_user_config", { clear = true })
+vim.api.nvim_create_autocmd("BufWritePost", {
+  command = "PackerCompile",
+  pattern = "plugins.lua",
+  group = group,
+})
 ```
 
-This autocommand can be placed in your `init.vim`, or any other startup file as per your setup.
-Placing this in `plugins.lua` could look like this:
-
-```lua
-vim.cmd([[
-  augroup packer_user_config
-    autocmd!
-    autocmd BufWritePost plugins.lua source <afile> | PackerCompile
-  augroup end
-]])
-```
+This autocommand can be placed in your `init.vim`, in `plugins.lua`, or any other startup file as per your setup.
```

Comment:
Should be `init.lua`, as this is Lua code.

Reply:
Good catch. However, I see that there are other references to `init.vim` in the readme. I guess it would be inconsistent if I used `init.lua` in this one place and the rest of the readme uses `init.vim`. I dropped the reference to `init.vim` entirely for now, I think that's probably the easiest solution.